setup/system.sh

```diff
@@ -144,6 +144,9 @@ pollinate  -q -r
 
 # Between these two, we really ought to be all set.
 
+echo 'Creating SSH public key... (might take a while)'
```

Comment:
My crypto friends say there's zero value in a longer key length, so I'm gonna go with them on this.